docs/source/miniconda.rst

```diff
@@ -21,8 +21,8 @@ Windows installers
    :header: Python version,Name,Size,SHA256 hash
    :widths: 5, 10, 5, 80
 
-   Python 3.9,`Miniconda3 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Windows-x86_64.exe>`_,57.7 MiB,``1ca0befe028325a766be4e3e0727b945``
-   ,`Miniconda3 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Windows-x86.exe>`_,54.9 MiB,``36cbe3b9a2933efa4ed71bb286368673``
+   Python 3.9,`Miniconda3 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Windows-x86_64.exe>`_,57.7 MiB,``c3a43d6bc4c4fa92454dbfa636ccb859a045d875df602b31ae71b9e0c3fec2b8``
+   ,`Miniconda3 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Windows-x86.exe>`_,54.9 MiB,``5045fb9dc4405dbba21054262b7d104ba61a8739c1a56038ccb0258f233ad646``
```

Comment:
Yup, those are what I get when I download them. (Sums were generated on Zeus, so just making sure they're the same as what is fetched.)